corehq/apps/accounting/models.py

```diff
@@ -1060,6 +1062,7 @@ def update_subscription(self, date_start=None, date_end=None,
             self.date_delay_invoicing = date_delay_invoicing
 
         self.do_not_invoice = do_not_invoice
+        self.no_invoice_reason = no_invoice_reason
```

Comment:
actually on second thought, since this follow the pattern of other fields in this function, best to leave as is for now.